src/ethereum/mod.rs

```diff
@@ -197,6 +233,24 @@ impl Ethereum {
         })
     }
 
+    /// Create contract instance
+    ///
+    /// # Arguments
+    ///
+    /// * `contract_address` -  The address of contract.
+    /// * `abi` - ABI of contract.
+    ///
+    /// # Returns
+    ///
+    /// Returns a `contract` instance.
+    pub fn contract_instance(&self, contract_address: Address, abi: &[u8]) -> Contract<Http> {
+        Contract::from_json(
+            self.web3.eth(),
+            H160::from(contract_address),
+            abi,
+        ).unwrap()
```

Comment:
We should handle errors unless they are part of "starting mosaic". Everything that can happen during "runtime" should be handled.
Is this function only called once on initialization or is it called repeatedly during execution?